src/ProgressOnderwijsUtils/NullableReferenceTypesHelpers.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using JetBrains.Annotations;
+
+namespace ProgressOnderwijsUtils
+{
+    public static class NullableReferenceTypesHelpers
+    {
+        [MustUseReturnValue]
+        public static T AssertNotNull<T>(this T? t)
+            where T : class
+            => t ?? throw new Exception("Reference is null!");
+
+        [MustUseReturnValue]
+        public static T? PretendNullable<T>(this T t)

```

Comment:
geen test voor deze?

Reply:
het was een vraag :)

Reply:
Ik lees tussen de regels door, als dat al mogelijk is met 1 regel.